pkg/apis/psp/OWNERS

```diff
@@ -0,0 +1,39 @@
+reviewers:
```

Comment:
I copied `OWNERS` file from `pkg/apis/extensions/OWNERS`. It it OK or we should start an empty `OWNERS` file?

Reply:
that's a huge list. OWNERS should be more focused.

Reply:
May I just remove `OWNERS` files?

Reply:
If it does not exist, the everything from the parent dir is inherited. But having a handful or less of reviewers here makes certainly sense.

Reply:
@sttts Ok, thanks!

I decided to include only myself as a reviewer because I don't know who also want to be reviewer for this stuff.

@pweil- @simo5 @adelton @enj Let me know if you want to be reviewers and I'll add you to this list.

Reply:
Sure.  Please include @liggitt @tallclair as well.

Reply:
Thanks! Added.